model/QtiItemCompiler.php

```diff
@@ -57,25 +57,36 @@ class QtiItemCompiler extends taoItems_models_classes_ItemCompiler implements Se
      * @var string
      */
     const INSTANCE_ITEMRUNNER = 'http://www.tao.lu/Ontologies/TAOItem.rdf#ServiceQtiItemRunner';
-    
+
+    /**
+     * {@inheritDoc}
+     * @see taoItems_models_classes_ItemCompiler::compile()
```

Comment:
the return type should be here, because based on the parents I don't really see what could it be

Reply:
Addressed this in a seperate PR: https://github.com/oat-sa/extension-tao-item/pull/314/files